# LinePosition.msg

# Define the centroid for the yellow line
float32 yellow_x
float32 yellow_y
bool yellow_detected

# Define the centroid for the white line
float32 white_x
float32 white_y
bool white_detected

# Define the centroid for the green line
float32 green_x
float32 green_y
bool green_detected

# Define the centroid for the red line
float32 red_x
float32 red_y
bool red_detected
